Thing.h

```diff
@@ -197,7 +195,7 @@ class ThingAction {
     // 2.11 Action object: A links array (An array of Link objects linking
     // to one or more representations of an Action resource, each with an
     // implied default rel=action.)
-    JsonArray inline_links = obj.createNestedArray("links");
+    JsonArray inline_links = obj.createNestedArray("forms");
```

Comment:
If I've understood correctly, you should also rename variable names from "links" to "forms".

Reply:
Ohh well, I guess I forgot to cherry pick that commit. Will do.